exchanges/src/types/direct.rs

```diff
@@ -148,6 +147,7 @@ mod tests {
 
     #[test]
     fn duplicate_queue_different_routing_bind_test() {
+        //TODO: test must be adjusted, len of bound_queue_names != bind count
```

Comment:
tests are needed to be done in the same PR

Reply:
okey boomer